examples/restored/README.md

```diff
@@ -0,0 +1,7 @@
+# Restored DB example
+
+An example that uses the module's default variable values. This example uses the IBM Cloud terraform provider to:
```

Comment:
"An example that uses the module's default variable values." -> not sure this makes sense here. It's not using the default variable. Suspect this was copy-pasted by mistake?

Reply:
Perhaps add a sentence that this example does not really match a real world scenario, but simply provide an end to end executable flow how a Postgresql database can be created from a backup. It's unlikely that anyone will want to create a postgres instance and then immediately a second one from a backup of the first one (which would be empty in most case)